packages/opencensus-instrumentation-ioredis/test/test-ioredis.ts

```diff
@@ -0,0 +1,202 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CoreTracer, logger, RootSpan, Span, SpanEventListener, SpanKind} from '@opencensus/core';
+import * as assert from 'assert';
+import * as ioredis from 'ioredis';
+
+import {plugin} from '../src';
+
+/** Collects ended root spans to allow for later analysis. */
+class RootSpanVerifier implements SpanEventListener {
+  endedRootSpans: RootSpan[] = [];
+
+  onStartSpan(span: RootSpan): void {
+    return;
+  }
+  onEndSpan(root: RootSpan) {
+    this.endedRootSpans.push(root);
+  }
+}
+
+/**
+ * Asserts root spans attributes.
+ * @param rootSpanVerifier An instance of rootSpanVerifier to analyse RootSpan
+ * instances from.
```

Comment:
Nit: could you indent `instances from.` by 4 spaces to make it visually clear that it's a continuation of the JS doc `@param` tag?